OpenRA.Game/World.cs

```diff
@@ -387,6 +387,7 @@ public void Dispose()
 
 			frameEndActions.Clear();
 
+			Sound.StopAudio();
```

Comment:
I don't think we have local controls/wrappers over sounds, so that shall stay, yes.